plansys2_domain_expert/include/plansys2_domain_expert/Types.hpp

```diff
@@ -104,7 +104,7 @@ class Assignment
    * @param[in] expression a bracketed e
    * @return a vector with the first level elements in the expression
    */
-  std::vector<std::string> splitExpr(const std::string & expression);
+  static std::vector<std::string> splitExpr(const std::string & expression);
```

Comment:
It is not necessary to convert it to `static` to check it. Create an `Assignment` object using the constructor with no parameters, and call the method (it's public).

Reply:
Ok, but isn't it useful to have this function `static` since it do not manipulate any class member or method ?
It can be used as an utility function without creating an `assignment`. Is this another place to put it or another function that already have the same behavior ?

Reply:
Maybe we should extract it as an external function in the same file.